content/en/docs/v3/_index.md

```diff
@@ -50,7 +50,7 @@ aliases:
         <h1 class="card-title">
           <a href="/docs/v3/develop/create-project/" title="create quickstarts or import new repositories">Create Projects</a>          
         </h1>
-        <p class="card-text align-bottom">create new quickstarts or import existing repositories into Jenkins XS</p>
+        <p class="card-text align-bottom">create new quickstarts or import existing repositories into Jenkins X</p>
```

Comment:
You want to wait for the versioning PR to be merged, and then rebase? `docs/v3` will be replaced by `docs-v3`.

Reply:
@ankit This MR can  be merged now, I think.

Reply:
Thank you, Ankit and Nitin :) 

Closing now in favour of #3135